phoenix-shared/src/commonMain/sqldelight/paymentsdb/fr/acinq/phoenix/db/sqldelight/Contacts.sq

```diff
@@ -0,0 +1,36 @@
+import fr.acinq.lightning.utils.UUID;
+import fr.acinq.phoenix.data.ContactInfo;
+
+CREATE TABLE IF NOT EXISTS contacts (
+     id BLOB AS UUID NOT NULL PRIMARY KEY,
+     data BLOB AS ContactInfo NOT NULL,
+     created_at INTEGER NOT NULL,
+     updated_at INTEGER DEFAULT NULL
+);
+
+listContacts:
+SELECT *
```

Comment:
If it just returns `data` then `ContactsQueries.listContacts()` can become : 
```kotlin
return queries.listContacts().executeAsList()
```

and `monitorContactsFlow` becomes :

```kotlin
return queries.listContacts().asFlow().mapToList(context)
```

Reply:
Done in [afe3283](https://github.com/ACINQ/phoenix/pull/640/commits/afe3283811e37018bebbe5132620ea0e452cf66e)